mockiato-codegen/src/parse/mockable_attr.rs

```diff
@@ -11,11 +12,15 @@ pub(crate) struct MockableAttr {
     /// The name sub-attribute. Example: `#[name = "FooMock"]`
     /// This customizes the name of the generated mock struct.
     pub(crate) name_attr: Option<NameAttr>,
+    /// The static sub-attribute. Example: `#[mockable(static)]`.
+    /// Enforces that only static lifetimes are used within the mock.
+    pub(crate) static_attr: Option<StaticAttr>,
```

Comment:
When writing `#[mockable(static)]`, I intuitively think of static objects. Could we rename this sub-attribute to something like `static_references`?

Reply:
Addressed in 9709405